src/app.scss

```diff
@@ -1,5 +1,6 @@
 /* Write your global styles here, in SCSS syntax. Variables and mixins from the src/variables.scss file are available here without importing */
-@use "uswds-fonts";
-@use "uswds-global";
-@use "uswds-typography";
-@use "uswds-utilities";
+@use "uswds" with (
+  $theme-font-path: $theme-font-path,
+  $theme-image-path: $theme-image-path,
+  $theme-show-notifications: false
+);
```

Comment:
Now that this is just a global import, we don't need the workaround with vite injecting it via `css.preprocessorOptions.scss.additionalData`